### DISCUSSION TITLE
Can not install package @jbrowse/plugin-sv-inspector

### DISCUSSION THREAD

**Discussion by BUGHERE** [GMOD/jbrowse-components#3619] [Q&A] [answered]
I am trying to a show sv-inspector view by writing code, and I noticed there is a package named "@jbrowse/plugin-sv-inspector" on this package.json:
![image](https://user-images.githubusercontent.com/55886903/229123039-9913f185-87db-4a39-b36f-2a36a142eeb4.png)

So I think i need to install package "@jbrowse/plugin-sv-inspector"(just like I need to install @jbrowse/react-linear-genome-view to show linear gemene view ). But i install fail, it seems like there are not this package in regitory:
![image](https://user-images.githubusercontent.com/55886903/229122739-ae7dffae-763d-4292-ac7c-6d67c47c782d.png)

**Comment by cmdcolin** [comment, marked as answer]
the 'sv inspector' is not available on NPM currently, it is only available as part of the 'jbrowse-web' app (which also isn't npm installable)



we do have a ongoing effort to publish more of our view types to NPM but currently only 'circular view' (@jbrowse/react-circular-genome-view) and 'linear genome view' (@jbrowse/react-linear-genome-view) are really usable

we will continue to try to get more view types published on NPM and may even be able to publish the full jbrowse-web app as an npm installable component (demo of this here https://github.com/GMOD/jbrowse-components/pull/3531)

**Reply by cmdcolin** [reply]
to be clear, the package name '@jbrowse/plugin-sv-inspector' is a 'private' package that is used by the jbrowse-web app in our monorepo, it wouldn't really be automatically reusable if we did publish it to npm as-is
